packages/theme/stories/introduction.stories.mdx

```diff
@@ -69,33 +69,34 @@ The styling of our UI Toolkit componentry is defined in two places, which operat
 
 Foundational tokens can broken into the following groups:
 
-| Foundation name | Description                                             | Example token                                             |
-| --------------- | ------------------------------------------------------- | --------------------------------------------------------- |
-| Color           | The core set of colors available                        | `-uitk-color-grey-800: rgb(36, 37, 38);`                  |
-| Shadow          | Shadows conveying elevation or hierarchy of components  | `--uitk-shadow-black-4: 0 6px 10px 0 rgba(0, 0, 0, 0.2);` |
-| Size            | Sizes in px to be used for general styling              | `--uitk-size-basis-unit: 4px;`                            |
-| Spacing         | Sizes in px used for calculating margins, paddings, etc | spacing units are influenced by `density`, examples below |
-| Typography      | General tokens relating to typography styles            | `--uitk-typography-weight-regular: 400;`                  |
-| Z-index\*       | Number indicating appropriate z-index of components     | `--uitk-zindex-tooltip: 1500;`                            |
+| Foundation name | Description                                            | Example token                                             |
+| --------------- | ------------------------------------------------------ | --------------------------------------------------------- |
+| Color           | The core set of colors available                       | `--uitk-color-grey-800: rgb(36, 37, 38);`                 |
+| Delay           | Durations between user actions and system feedback     | `--uitk-delay-cutoff: 10000;`                             |
+| Opacity         | Opacity percentages for conveying disabled states      | `--uitk-opacity-background: 0.7;`                         |
```

Comment:
Not sure putting "disabled" in the opacity description is good or not. Opacity can potentially mean other things, our disabled characteristics just happen to choose opacity.